### PR TITLE
Show tournament winners (part 1)

### PR DESCRIPTION
**Pull request by ErikBrendel**
The winner and second place of a tournament get displayed.

Displaying place three is still in progress, and will come with a second PR.

### CODE REVIEW

File: app/views/events/show.html.erb [hunk 1]
```diff
@@ -22,6 +22,27 @@
   <% end %>
 </div>
 
+<% if @event.is_a? Tournament and @event.finale.present? and @event.finale.has_winner? %>
+  <div class="table-responsive">
+    <h3><%= t('events.show.winners') %></h3>
+    <table class="table">
+      <tr>
+        <th>Platzierung</th>
+        <th><%= @event.single? ? t('.title', default: User.model_name.human.titleize) : t('.title', default: Team.model_name.human.titleize)%></th>
```
Comment (Markus-Brand) [inline app/views/events/show.html.erb]: Is there no better way to do this? looks kinda ugly :)

**Review by Markus-Brand — APPROVED** [review]

File: app/views/events/show.html.erb (hunk 1 above)
Comment (fpottbaecker) [reply app/views/events/show.html.erb]: You could use `@event.finale.winner.model_name.human.titleize` (or any participant for that matter.

File: app/views/events/show.html.erb [hunk 2]
```diff
@@ -58,7 +79,7 @@
       <thead><tr><th><%=t '.title', default: User.model_name.human.titleize %></th></tr></thead>
       <tbody>
         <% @event.participants.each do |participant| %>
-          <tr><td><%= participant.name %></td></tr>
+          <tr><td><%= link_to participant.name, user_path(participant), id: participant.name %></td></tr>
```
Comment (fpottbaecker) [inline app/views/events/show.html.erb]: Shouldn't this use event participant link?

File: app/helpers/events_helper.rb [hunk 3]
```diff
@@ -1,2 +1,6 @@
 module EventsHelper
+  def event_participant_link(participant)
```
Comment (fpottbaecker) [inline app/helpers/events_helper.rb]: Rename this to `link_to_participant`?

File: features/tournament/show_winner.feature [hunk 4]
```diff
@@ -0,0 +1,14 @@
+Feature: Show Tournament winners
+  As a tournament winner
+  I want to be shown
+  In order to let everyone know that my team is awesome
+
+  Scenario: a small Tournament gets finished
+    Given a tournament t with 4 teams
+    And the Spielplan page for t is visited
+    And the results for match Halbfinale 1 (3 : 4) got inserted
```
Comment (fpottbaecker) [inline features/tournament/show_winner.feature:9]: have been entered?

File: features/tournament/show_winner.feature [hunk 5]
```diff
@@ -0,0 +1,14 @@
+Feature: Show Tournament winners
+  As a tournament winner
+  I want to be shown
+  In order to let everyone know that my team is awesome
+
+  Scenario: a small Tournament gets finished
+    Given a tournament t with 4 teams
+    And the Spielplan page for t is visited
```
Comment (fpottbaecker) [inline features/tournament/show_winner.feature:8]: schedule?

File: features/step_definitions/tournament.rb [hunk 6]
```diff
@@ -96,3 +96,14 @@ def find_team_of_match(match_gameday, match_num, home_or_away)
 Given(/^a tournament with gamemode (.*)$/) do |mode|
```
Comment (fpottbaecker) [inline features/step_definitions/tournament.rb:96]: While a little bit off for this pr, you should actually use the except syntax [there](https://github.com/hpi-swt2/sport-portal/pull/274/files#diff-1b3862269b88b8c332be42cbd071b46eR79).

**Review by fpottbaecker — CHANGES_REQUESTED** [review]

**Review by fpottbaecker — APPROVED** [review]

**Pull request merged**